tests/sharding/fixtures.py

```diff
@@ -25,11 +30,6 @@
     get_default_account_keys,
 )
 
-from eth_utils import (
-    to_canonical_address,
-    to_checksum_address,
-)
-
 from evm.vm.forks.spurious_dragon.transactions import (
     SpuriousDragonTransaction,
```

Comment:
Just curious, why use `SpuriousDragonTransaction` instead of `ByzantiumTransaction`?

Reply:
Thank you for pointing out!
I will change it.